EIPS/eip-draft_DAOs.md

```diff
@@ -0,0 +1,240 @@
+
+### **Community Consensus**
+
+The initial draft standard was developed as part of the DAOstar One roundtable series, which included representatives from all major EVM-based DAO frameworks (Aragon, Compound, DAOstack, Gnosis, Moloch, OpenZeppelin, and Tribute), a wide selection of DAO tooling developers, as well as several major DAOs. Thank you to all the participants of the roundtable, the full list of which can be found [here](https://daostar.one/). We would especially like to thank Auryn Macmillan, Fabien of Snapshot, Selim Imoberdorf, Lucia Korpas, and Mehdi Salehi for their contributions.
+
+In-person events will be held at Schelling Point 2022 and at ETHDenver 2022, where we hope to receive more comments from the community. We also plan to schedule a series of community calls through early 2022.
+
+## Reference Implementations
+
+A number of basic [reference implementations](https://github.com/metagov/daostar) have been published, including ones for Moloch, DAOstack, and Compound.
+
+## Security Considerations
+
+This standard defines the interfaces for the DAO URIs but does not specify the rules under which the URIs are set, or how the data is prepared. Developers implementing this standard should consider how to update this data in a way aligned with the DAO’s governance model, and keep the data fresh in a way that minimizes reliance on centralized service providers.
+
+Indexers that rely on the data returned by the URI should take caution if DAOs return executable code from the URIs. This executable code might be intended to get the freshest information on membership, proposals, and activity log, but it could also be used to run unrelated tasks.
+
+## History
+
+## References
+
+1. Dilger, W. (1997). Decentralized autonomous organization of the intelligent home according to the principle of the immune system’. *1997 IEEE International Conference on Systems, Man, and Cybernetics. Computational Cybernetics and Simulation*, 351–356. [https://doi.org/10.1109/ICSMC.1997.625775](https://doi.org/10.1109/ICSMC.1997.625775)
+2. Buterin, V. (2013a). *Ethereum whitepaper: A next-generation smart contract and decentralized application platform* [White Paper]. [https://blockchainlab.com/pdf/Ethereum_white_paper-a_next_generation_smart_contract_and_decentralized_application_platform-vitalik-buterin.pdf](https://blockchainlab.com/pdf/Ethereum_white_paper-a_next_generation_smart_contract_and_decentralized_application_platform-vitalik-buterin.pdf)
+3. JSON-LD Schema. [https://json-ld.org/](https://json-ld.org/)
+4. CAIP-10. [https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-10.md](https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-10.md)
+
```

Comment:
```suggestion
```
This is not a valid EIP section and should be removed.  Plus external links should not be in an EIP.  Feel free to include them in the discussions-to link.